cilium-health/launch/endpoint.go

```diff
@@ -320,7 +318,15 @@ func LaunchAsEndpoint(baseCtx context.Context,
 	}
 
 	// Set up the endpoint routes.
-	if err = configureHealthRouting(info.ContainerName, epIfaceName, node.GetNodeAddressing(), mtuConfig); err != nil {
+	routes, err := getHealthRoutes(node.GetNodeAddressing(), mtuConfig)
+	if err != nil {
+		return nil, fmt.Errorf("Error while configuring routes for containername %q: %s", info.ContainerName, err)
```

Comment:
nit: `getHealthRoutes()` doesn't configure, it just calculates, right? This log line is a bit misleading if so.